docs/fedora-releases-guide/what-to-know.md

```diff
@@ -0,0 +1,106 @@
+
+There is an [open issue](https://github.com/packit/packit-service/issues/2404) for Bodhi update notifications.
+
+:::
+
+## Monitoring and observability
+
+Track *Packit Service* activity through:
+- the pull request updates, and 
+- the [Packit Dashboard](https://dashboard.packit.dev/).
+
+Packit Service responds to every valid command in pull requests. Missing response indicates:
+- Incorrect command syntax
+- Packit Service outage
+
+Check service status at [our status page](https://status.packit.dev/).
+
+## Recovery procedures
+
+### Service-based retriggering
+
+Each job configured in `packit.yml` can be retriggered via dist-git pull request comments. See the [complete list of commands](https://packit.dev/docs/fedora-releases-guide/dist-git-onboarding#retriggering).
+
+### Manual CLI Operations
+
+During service outages, maintain workflow using Packit CLI:
+
+1. Install [Packit CLI](https://packit.dev/docs/cli)
+2. [Configure required tokens](https://packit.dev/docs/configuration#user-configuration-file)
+3. Verify setup (if not already working in service):
+   - Follow [dist-git onboarding guide](https://packit.dev/docs/fedora-releases-guide/dist-git-onboarding)
+   - Validate config with `packit validate-config`
+
+#### Troubleshooting Issues
+
+**Missing pull request for release:**
+
+Use [`packit pull-from-upstream`](https://packit.dev/docs/cli/pull-from-upstream) when [release-monitoring](https://release-monitoring.org/) detects updates but pull requests aren't created.
+
+**Missing Koji build:**
+
+The release pull request is merged (and contains a change in the version value) but a Koji build isn't triggered.
+
+- Check [`allowed_pr_authors`](https://packit.dev/docs/configuration/downstream/koji_build#optional-parameters)
+- Trigger build manually with [`packit build in-koji`](https://packit.dev/docs/cli/build/in-koji) (and specify dist-git branches if needed)
+
+When using Packit CLI to create pull requests, ensure your FAS account is added to the `allowed_pr_authors` list to trigger Koji builds automatically. This enables the expected build workflow for maintainer-created pull requests.
+
+**Missing Bodhi update:**
+
+The Koji build is created but there is no Bodhi update.
+
+- Packit does not create Bodhi updates for `rawhide`, they are automatically created by Bodhi.
+- Verify [`allowed_builders`](https://packit.dev/docs/configuration/downstream/bodhi_update#optional-parameters)
+- Create manually using [`packit create-update`](https://packit.dev/docs/cli/create-update)
+
+For updates on non-rawhide branches, verify your FAS account is included in the `allowed_builders` list. When submitting Koji builds via Packit CLI, automatic Bodhi update creation requires your FAS account to be explicitly authorized in this configuration.
+
+## Customization options
+
+Packit works best with simple package structures:
+- Single source
+- Single patch
+- Straightforward spec file
```

Comment:
I am not sure this is accurate, e.g. as for sources, I would say currently, with only remote sources (multiple) Packit works ok out of the box; also not sure what is the exact situation or patches, but I could see some packages having them and Packit working fine. As for the specfiles, "straightforward" might also not be that clear, maybe we could mention concrete issues, like specfile parsing doesn't handle conditions or using some extra macros, @nforro what do you think?

Reply:
Ok! I took this from the script that was checking the projects suitable to be onboarded.  Are they working fine without custom hooks or actions? As suggested in the following sentence?

Reply:
I don't think there are any problems with sources anymore, condition parsing is not really a concern during sync_release, but even if it was, the long-standing issue in specfile is only about conditions outside of sections or spanning across sections boundaries, which is quite rare. As for downstream patches, I think the situation is still the same - if they have been upstreamed they will fail to apply to new version sources.

Reply:
Ok, thanks for the details, I would then probably sum it e.g. like `Packit works out of the box with the simple package structures (e.g. non-complex spec files), but can be customised for more complicated use cases as well.`

Reply:
@majamassarini oh I see, I think since the script was created, we have changed quite a lot regarding the source handling, so we now support bigger variety of packages.

Reply:
I would maybe change it to `Packit should work out of the box with the simple package structures ...`, just to be on the safe side.